src/App.tsx

```diff
@@ -410,6 +415,44 @@ function App() {
                   </div>
                 </div>
 
+                {/* Collapsible section for organizer email */}
+                <CollapsibleSection
```

Comment:
It doesn't need to be a collapsible. One field, with an (i) in front of it is enough.

(Note: The (i) needs to be touch friendly)

Reply:
Plz read https://github.com/omid/calf/pull/1#discussion_r2382392122 also